rpm/README.md

```diff
@@ -0,0 +1,43 @@
+# System Admin Toolkit Offline Documentation
+
+The SAT documentation can be found online at the following url:
+https://cray-hpe.github.io/docs-sat
+
+Additionally, it is available offline in markdown format, which may be viewed
+with a markdown viewer or simply with a text editor. A table of contents can be
+found below.
+
+The offline documentation is available in the `docs/` directory of the SAT
+release distribution as well as in RPM package format. The RPM package is
+installed as a part of the Ansible plays launched by the Configuration
+Framework Service (CFS). Its files are installed to `/usr/share/doc/sat`.
+
+## Check for Latest Documentation
+
+Acquire the latest documentation RPM. This may include updates, corrections,
+and enhancements that were not available until after the software release.
+
+1. Check the version of the currently installed SAT documentation. This RPM
```

Comment:
Nite: could you move the second sentences in each of these steps into the step body? Looks cleaner to me to keep the step titles to one sentence each.

Reply:
Fixed with 22d69ae182d7bcc288d76a4bc0afb2848ee9ea29